logging/options.go

```diff
@@ -31,6 +31,12 @@ type Options struct {
 	// meaning that logfmt output is used.
 	JSON bool `json:"json"`
 
+	// FMTType is to change the output style. The default is "term", for ease of use for debuging.
+	// Another option is "fmt", for plain txt output
+	FMTType string `json:"fmttype"`
+
+	TermOptions TextFormatter `json:"term_options"`
```

Comment:
Don't use underscores, per https://golang.org/doc/effective_go.html#names

`json:"termOptions"` is preferred.